vNext/test/TesterInternal/TesterInternal.csproj

```diff
@@ -51,7 +51,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="..\..\..\test\TesterInternal\**\*.cs" Exclude="..\..\..\test\TesterInternal\obj\**\*.cs;..\..\..\test\TesterInternal\bin\**\*.cs;..\..\..\test\TesterInternal\Properties\*.cs;..\..\..\test\TesterInternal\CounterControlProgTests.cs;..\..\..\test\TesterInternal\GeoClusterTests\*.cs;..\..\..\test\TesterInternal\HostedTestClusterBase.cs;..\..\..\test\TesterInternal\OrleansHostProgTests.cs;..\..\..\test\TesterInternal\General\BootstrapProviderTests.cs;..\..\..\test\TesterInternal\StorageTests\PersistenceGrainTests.cs;..\..\..\test\TesterInternal\StreamingTests\StreamPubSubReliabilityTests.cs" />
+    <Compile Include="..\..\..\test\TesterInternal\**\*.cs" Exclude="..\..\..\test\TesterInternal\obj\**\*.cs;..\..\..\test\TesterInternal\bin\**\*.cs;..\..\..\test\TesterInternal\Properties\*.cs;..\..\..\test\TesterInternal\CounterControlProgTests.cs;..\..\..\test\TesterInternal\GeoClusterTests\*.cs;..\..\..\test\TesterInternal\HostedTestClusterBase.cs;..\..\..\test\TesterInternal\OrleansHostProgTests.cs" />
```

Comment:
I believe HostedTestClusterBase.cs was removed, so no need to exclude it any more either. Could be cleaned up in a separate PR if you prefer. I don't feel strongly about it.